x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/value_lists/value_list_items.cy.ts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { login } from '../../../../tasks/login';
+import { visit } from '../../../../tasks/navigation';
+import { TOASTER_BODY } from '../../../../screens/alerts_detection_rules';
+import { closeErrorToast } from '../../../../tasks/alerts_detection_rules';
+import {
+  createListsIndex,
+  waitForValueListsModalToBeLoaded,
+  openValueListsModal,
+  importValueList,
+  waitForListsIndex,
+  deleteValueLists,
+  KNOWN_VALUE_LIST_FILES,
+  openValueListItemsModal,
+  searchValueListItemsModal,
+  getValueListItemsTableRow,
+  checkTotalItems,
+  deleteListItem,
+  clearSearchValueListItemsModal,
+  sortByValue,
+  updateListItem,
+  addListItem,
+  selectValueListsItemsFile,
+  uploadValueListsItemsFile,
+  mockFetchListItemsError,
+  mockCreateListItemError,
+  mockUpdateListItemError,
+  mockDeleteListItemError,
+} from '../../../../tasks/lists';
+import {
+  VALUE_LIST_ITEMS_MODAL_INFO,
+  VALUE_LIST_ITEMS_MODAL_TABLE,
+  VALUE_LIST_ITEMS_MODAL_TITLE,
+} from '../../../../screens/lists';
+import { RULES_MANAGEMENT_URL } from '../../../../urls/rules_management';
+
+describe(
+  'Value list items',
+  {
+    tags: ['@ess', '@serverless'],
```

Comment:
As per the [new labeling changes](https://github.com/elastic/kibana/pull/179737), this test should be marked as `@skipInServerless`. Currently on MKI environments there is no way for enabling automatically FF so this test is going to fail in the periodic pipeline. cc @yctercero